apps/preview/next/pages/showcases/SelectDropdown/SelectDropdownDisabled.tsx

```diff
@@ -0,0 +1,126 @@
+import { ShowcasePageLayout } from '../../showcases';
+// #region source
+import { useId, useRef, useState, type KeyboardEvent } from 'react';
+import classNames from 'classnames';
+import { SfIconExpandMore, SfListItem, SfDropdown, SfIconCheck, useTrapFocus } from '@storefront-ui/react';
+
+type OptionType = {
```

Comment:
It's a small thing, but don't you think including word `Type` in a _type_ definition is unnecessary? I mean, you know it's type, right? 😄 Maybe `SelectOption` instead?

Reply:
true, it was in the preview implementation

Reply:
76ea2c2fed6d6a6c3be649a99c4935a758160ec0